agent/workers/trigger.go

```diff
@@ -42,6 +62,10 @@ func (w *TriggerWorker) Trigger(ctx context.Context, triggerRequest *proto.Trigg
 		return fmt.Errorf("invalid traceID was received in TriggerRequest: %w", err)
 	}
 
+	// Set traceID to cache so the collector starts watching for incoming traces
+	// with same id
+	w.traceCache.Set(triggerRequest.TraceID, []*v1.Span{})
```

Comment:
I guess we don't know what ds is used at this point right?

Reply:
yeah, we don't :(